OSVR-Unity/Assets/OSVRUnity/src/VREye.cs

```diff
@@ -247,20 +249,31 @@ public void CreateSurfaces(uint surfaceCount)
 
                     //distortion
                     bool useDistortion = Viewer.DisplayController.DisplayConfig.DoesViewerEyeSurfaceWantDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
-                    if(useDistortion)
+                    useDistortion |= !displayController.UseRenderManager;
```

Comment:
We also want to apply this change here, right? https://github.com/OSVR/OSVR-Unity/blob/master/OSVR-Unity/Assets/OSVRUnity/src/VREye.cs#L204

When I debug the OSVRDemo2 scene, it doesn't reach this code.